entity-framework/core/querying/related-data.md

```diff
@@ -51,6 +51,27 @@ You may want to include multiple related entities for one of the entities that i
 > [!CAUTION]
 > Since version 3.0.0, each `Include` will cause an additional JOIN to be added to SQL queries produced by relational providers, whereas previous versions generated additional SQL queries. This can significantly change the performance of your queries, for better or worse. In particular, LINQ queries with an exceedingly high number of `Include` operators may need to be broken down into multiple separate LINQ queries in order to avoid the cartesian explosion problem.
 
+### Filtered include
+
+> [!NOTE]
+> This feature is introduced in EF Core 5.0.
+
+The filtered include feature enables applying operations directly on the included collection navigation, which allows for filtering and sorting of the results.
+
+Supported operations are: `Where`, `OrderBy`, `OrderByDescending`, `ThenBy`, `ThenByDescending`, `Skip`, and `Take`.
+
+Those operations should be applied inside the Include method:
+
+[!code-csharp[Main](../../../samples/core/Querying/RelatedData/Sample.cs#FilteredInclude)]
+
+Each included navigation allows only one unique set of filter operations. In cases where multiple Include operations are applied on a given collection navigation (`blog.Posts` in the examples below), filter operations can only be applied to one of them: 
```

Comment:
```suggestion
Each included navigation allows only one unique set of filter operations. In cases where multiple Include operations are applied for a given collection navigation (`blog.Posts` in the examples below), filter operations can be specified on one of them:
```

Reply:
We should just merge both of these in one paragraph. Also it is not like all or none. you can specifying the filter on any occurrence of collection navigation as long as it is same.

Reply:
I read your comment above, if you think lower priority and we should keep separate then it is fine. Though consider above suggestion.